verification.py

```diff
@@ -272,10 +272,25 @@ def f():
             self._run_mutation_stats(key, command_line)
         self._instances[key] = f
 
-    def _run_mutation_discoal_stats(self, key, args):
+    def _discoal_str_to_ms(self, args):
         # convert discoal string to msprime string
         tokens = args.split(" ")
-        msp_str = " ".join(tokens[0:2]+tokens[3:])
+        # cut out sites param
```

Comment:
Not sure what this is doing @andrewkern - is the idea that we're creating an ms-like command line for msprime to run? If so we need to take a different approach I think, as I'm not planning on adding support for sweeps to the mspms interface (it's already super complicated, and I don't want to touch it unless absolutely necessary).

Reply:
yes that's the idea-- i'm translating a `discoal` string to `ms`. how should i proceed instead? btw these checks that i added all looked good to me, so at least `msprime` and `discoal` are giving similar results for these neutral examples